packages/bot-skeleton/src/services/tradeEngine/utils/helpers.js

```diff
@@ -90,7 +91,7 @@ export const shouldThrowError = (error, errors_to_ignore = []) => {
         'RateLimit',
         'DisconnectError',
     ];
-    const is_ignorable_error = errors_to_ignore.concat(default_errors_to_ignore).includes(error.name);
+    const is_ignorable_error = errors_to_ignore.concat(default_errors_to_ignore).includes(error.code);
```

Comment:
Is it `error.code` or `error.error.code`? Just checking because that's the pattern above 😅

Reply:
Same answer 😄

Reply:
So `error.error.code` and `error.code` return the same? 🤔

Reply:
No, the same answer refer to the previous comment which I left on Carol comment 😂 
on our code, some of our errors return as response object(error.error.name) the other errors return as error object(error.name) but on updating the code I try to send all error in a similar way so we don't need this optional chaining

Reply:
The issue here is that on line e.g. line 70 you're checking `error.error.code === 'MarketIsClosed'` whereas on this line you're checking whether `error.code` is contained in the array. What is the object structured like?